snuba/clickhouse/query_dsl/accessors.py

```diff
@@ -189,7 +190,7 @@ def get_time_range(
                 ),
             ),
             (
-                Column(None, String(timestamp_field)),
+                TransformedColumn(Column(None, String(timestamp_field))),
```

Comment:
This is going to impact the query splitter. Could you please add a test to ensure that works properly? (snuba/tests/test_split.py). You should be able to add a case with a timestamp embedded in a function.

Reply:
I removed this for now. It can be done separately.